.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 Thumbs.db
 .idea/
+go.sum
```

Comment:
I'd rather not complexify the PR with adding module support. Can you remove this and make it in another PR?

Reply:
Removed module support, left for future PR